script/hassfest/requirements.py

```diff
@@ -90,7 +90,7 @@ def validate_requirements_format(integration: Integration) -> bool:
         if not version:
             continue
 
-        for part in version.split(","):
+        for part in version.split(";", 1)[0].split(","):
```

Comment:
Can break this out into another PR if desirable but left here initially for context

Reply:
This is OK I think 👍